Rakefile

```diff
@@ -15,7 +15,12 @@ Rake::TestTask.new(:test) do |test|
   test.pattern = 'test/**/*_test.rb'
 end
 
-task :default => [:compile, :test]
+# When runnig in Travis, build taglib first
+if ENV['TRAVIS'] == 'true'
```

Comment:
Not sure about doing it this way, can we do it in a "before script" or something in `.travis.yml` instead?

Reply:
Sure, that's a good idea.